poetry.toml

```diff
@@ -0,0 +1,2 @@
+[virtualenvs]
+in-project = true
```

Comment:
What does this file do?

Reply:
`poetry` by default tries to create all you virtual environments in your home directory, this just tells poetry to create the virtual environment in the project

Reply:
Neat!